action.yml

```diff
@@ -17,92 +17,58 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: 'Checkout codebase'
-      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      with:
-        lfs: 'true'
-        submodules: 'true'
-        fetch-depth: 0
-        fetch-tags: true
+    - name: Checkout codebase
+      uses: ./mobile-android-pipelines/actions/checkout-codebase
 
     - name: Clean workspace
       uses: ./mobile-android-pipelines/actions/clean-workspace
 
-    - name: 'Detect architecture'
-      id: detect-arch
-      uses: ./mobile-android-pipelines/actions/detect-arch
-
     - name: Setup GitHub Runner workflow
       uses: ./mobile-android-pipelines/actions/setup-runner
       
     - name: Verify Conventional commit standards against latest git tag
-      run: cog check -l
-      shell: bash
+      uses: ./mobile-android-pipelines/actions/verify-conventional-commit
```

Comment:
There's an official [cocogitto github action ](https://docs.cocogitto.io/ci_cd/action.html#github-action) that might be better to switch t, rather than rolling our own

Reply:
That's nice! Implemented